e2e/process-services-cloud/task-list/edit-process-filters-component.e2e.ts

```diff
@@ -192,9 +179,9 @@ describe('Edit process filters cloud', () => {
         const dialog = editProcessFilter.editProcessFilterDialog();
 
         await expect(await dialog.checkCancelButtonIsEnabled()).toEqual(true);
-        await expect(await dialog.checkSaveButtonIsEnabled()).toEqual(true);
+        await expect(await dialog.checkSaveButtonIsEnabled()).toEqual(false);
         await expect(await dialog.getTitle()).toEqual('Save filter as');
-        await expect(await dialog.getFilterName()).toEqual(PROCESSES.ALL);
```

Comment:
Sorry, I know it's not part of your PR but in case you do another commit can you please rename this variable to PROCESS_FILTERS instead of PROCESSES?

Reply:
I will for sure do other commit and i will change it, thanks.